Simplenote/SPAppDelegate.m

```diff
@@ -182,8 +184,11 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
 
     [self showPasscodeLockIfNecessary];
     [self cleanupScrollPositionCache];
-    [self syncWidgetDefaults];
-    [self resetWidgetTimelines];
+
+    if (@available(iOS 14.0, *)) {
+        [self syncWidgetDefaults];
+        [self resetWidgetTimelines];
+    } 
```

Comment:
Suggestion: Rather than safeguarding every spot that calls `syncWidget` / `resetWidget`, you can just safeguard at the API level:

```
    @objc
    func resetWidgetTimelines() {
        guard #available(iOS 14.0, *) else {
            return
        }

        WidgetController.resetWidgetTimelines()
    }
```

Or even better, you can just add that snippet in `resetWidgetTimelines`. That shields you from any accidental bug like this, in the future.